docs/content/v1.0.x/release-notes/_index.md

```diff
@@ -5,6 +5,40 @@ menu:
 docs:
 weight: 100
 ---
+### v1.0.8
+Add more detailed message if generation fails.
+
+Fix setting child type when generating parent type.
+
+### v1.0.7
+Fix fixture-monkey-kotlin module supporting Jdk 8.
+
+### v1.0.6
+Fixture Monkey will now infer the properties when you generate a type by specific ArbitraryIntrospector that needs certain type of properties.
```

Comment:
It would be better if you change class names like  ArbitraryIntrospector  to ` ArbitraryIntrospector ` for better readability.